packages/tre/src/plugins/cache/gateway.ts

```diff
@@ -207,11 +209,12 @@ export class CacheGateway {
     private readonly clock: Clock
   ) {}
 
-  async match(request: Request): Promise<Response> {
+  async match(request: Request, cacheKey?: string): Promise<Response> {
```

Comment:
I could be wrong, but I don't believe `workerd` ever sends a `cacheKey`—I think the request URI should always be treated as the key?